### ISSUE TITLE
Getting cwltool warnings when running bamstats

### ISSUE THREAD

**Issue by coverbeck** [dockstore/dockstore#4819]
**Describe the bug**
I noticed some cwltool warnings when running bamstats. We use this tool in our docs/tutorials (I think), so it should ideally be a clean experience for users following along.

**To Reproduce**
I activate a Python virtual environment 3.9.4 to begin with

Save below as a file and run 
```
curl -o requirements.txt "https://staging.dockstore.org/api/metadata/runner_dependencies?client_version=1.12.0&python_version=3"
pip install -r requirements.txt
cwltool --version

cat <<EOT >> Dockstore.json
{
  "mem_gb": 4,
  "bam_input": {
    "path": "https://github.com/CancerCollaboratory/dockstore-tool-bamstats/raw/develop/rna.SRR948778.bam",
    "format": "http://edamontology.org/format_2572",
    "class": "File"
  },
  "bamstats_report": {
    "path": "/tmp/bamstats_report.zip",
    "class": "File"
  }
}
EOT

echo "On Mac, required 5GB memory allocated to Docker"
dockstore tool launch --entry quay.io/collaboratory/dockstore-tool-bamstats:1.25-6_1.0 --json Dockstore.json
```
cwltool  version will be 3.0.20200807132242 (comes from requirements.txt)

Tool runs successfully, but you see these warnings;

```
URI prefix 'foaf' of 'foaf:name' not recognized, are you missing a $namespaces section?
URI prefix 'foaf' of 'foaf:mbox' not recognized, are you missing a $namespaces section?
../../../var/folders/nr/dkx1p38s24nb7fx_qgcxggcm0000gn/T/1648687018453-0/Dockstore.cwl:7:1: Warning: invalid
                                                                                            field
                                                                                            `description`,
                                                                                            expected one of:
                                                                                            'id', 'inputs',
                                                                                            'outputs',
                                                                                            'requirements',
                                                                                            'hints', 'label',
                                                                                            'doc',
                                                                                            'cwlVersion',
                                                                                            'class',
                                                                                            'baseCommand',
                                                                                            'arguments',
                                                                                            'stdin', 'stderr',
                                                                                            'stdout',
                                                                                            'successCodes',
                                                                                            'temporaryFailCodes',
                                                                                            'permanentFailCodes'
Warning: invalid field `doc`, expected one of: 'class', 'coresMin', 'coresMax', 'ramMin', 'ramMax', 'tmpdirMin', 'tmpdirMax', 'outdirMin', 'outdirMax'
```

**Expected behavior**
It should run without warnings

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-2121)
┆fixVersions: Dockstore 1.13
┆friendlyId: DOCK-2121
┆sprint: Backlog
┆taskType: Story

**Comment by denis-yuen** [comment]
Oh, I see. I cwl itself has 
```
cwlVersion: v1.0 
description: |
    ![build_status](https://quay.io/repository/collaboratory/dockstore-tool-bamstats/status)
    A Docker container for the BAMStats command. See the [BAMStats](http://bamstats.sourceforge.net/) website for more information.
```
i.e. the error message is reversed. In cwl 1.0.2, description was renamed to doc and the cwltool message is just reversed.
Probably will be fixed by #4335 

Recommend closing since its just a misleading warning

**Comment by denis-yuen** [comment]
Confirmed warning messages are not present when pairing  `quay.io/collaboratory/dockstore-tool-bamstats:1.25-7` with `3.1.20220224085855` (version used for 1.13.0)
